src/olympia/devhub/tests/test_tasks.py

```diff
@@ -567,7 +567,7 @@ def test_run_linter_use_temporary_file_large_output(self):
 
             assert tmpf.call_count == 2
             assert result['success']
-            assert result['warnings'] > 700
+            assert result['warnings'] == 11
```

Comment:
This needed to change since the behavior in the linter changed, it's just asserting basic functionality so nothing special here